modules/Makefile

```diff
@@ -26,7 +26,7 @@ OCFDIR=$(PWD)/../ocf
 PWD=$(shell pwd)
 KERNEL_VERSION ?= "$(shell uname -r)"
 KERNEL_DIR ?= "/lib/modules/$(KERNEL_VERSION)/build"
-MODULES_DIR=/lib/modules/$(shell uname -r)/extra
+MODULES_DIR=/lib/modules/$(shell uname -r)/extra/block/opencas
```

Comment:
use $KERNEL_VERSION instead of hardcoding "uname -r"

Reply:
done